resources/views/technique/index.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.app')
+@section('title', $viewData['title'])
+@section('content')
+@foreach ($viewData['technique'] as $technique) 
+<div class="col-md-4 col-lg-3 mb-2" style="margin: 15px;"> 
+    <div class="card" style="width: 18rem;"> 
+        <img src="{{ $technique['image'] }}" class="card-img-top" alt="{{ $technique['model'] }}"> 
+        <div class="card-body"> 
+            <h5 class="card-title">
+                <a href="{{ route('technique.show', ['id'=> $technique['id']]) }}">{{ $technique->getModel() }}</a>
```

Comment:
Lo cambiaría a esto (para utilizar solo getters)
```suggestion
                <a href="{{ route('technique.show', ['id'=> $technique->getId()]) }}">{{ $technique->getModel() }}</a>
```